src/store/modules/search.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios'
+import { path } from '../../api/tmdb-api'
```

Comment:
Instead of `../../` you can use `@`. The `@` is a shorthand for `src` folder.
Final string will looks like: `@/api/tmdb-api`